### DISCUSSION TITLE
No luck getting Nominatim running on Ubuntu 20.04

### DISCUSSION THREAD

**Discussion by smartsite** [osm-search/Nominatim#3483] [Q&A]
I ran through this guide:

https://nominatim.org/release-docs/latest/appendix/Install-on-Ubuntu-20/

and I completed all the steps without any errors reported. If I try and access the API via localhost, I first get a proxy error, then if I hit f5 I get **Service Unavailable** each time after. 

After digging through logs, it's borking at this point:

/path/to/nominatim-venv/bin/gunicorn -w 4 -k uvicorn.workers.UvicornWorker nominatim.wsgi:application

This is also displayed when I try and start the service with: 

sudo systemctl status nominatim

If I attempt to run it via uvicorn, I also get "Could not import module "nominatim.wsgi".

Any idea what is going on? Thanks in advance.

**Comment by mtmail** [comment]
> This is also displayed when I try and start the service with:
> sudo systemctl status nominatim

Can you post the output? Maybe you forgot to upload a screenshot (though text-only is preferred)?

**Comment by smartsite** [comment]
![image](https://github.com/user-attachments/assets/0296065f-49e6-42f0-87c5-672c6f6b1444)

**Comment by lonvia** [comment]
Which part of the documentation tells you to run `nominatim.wsgi:application`?

**Reply by smartsite** [reply]
That is me executing the line of the script where the error was reported to confirm it. Do you have a recommendation?

**Reply by lonvia** [reply]
The documentation you have linked, tells you to run:

    /path/to/nominatim-venv/bin/gunicorn -w 4 -k uvicorn.workers.UvicornWorker nominatim.server.falcon.server:run_wsgi
